backend/app/tests/crud/config/test_config.py

```
@@ -0,0 +1,476 @@
+    assert len(configs_page2) == 2
+    assert configs_page1[0].id != configs_page2[0].id
+
+
+def test_read_all_configs_ordered_by_updated_at(db: Session) -> None:
+    """Test that configurations are ordered by updated_at in descending order."""
+    project = create_test_project(db)
+    config_crud = ConfigCrud(session=db, project_id=project.id)
+
+    # Create configs (they will have different updated_at timestamps)
+    config1 = create_test_config(db, project_id=project.id, name="config-1")
+    config2 = create_test_config(db, project_id=project.id, name="config-2")
```

Ruff (0.14.4)

Local variable `config2` is assigned to but never used

Remove assignment to unused variable `config2`

(F841)